src/dev/economy.rb

```diff
@@ -63,11 +63,11 @@ def self.get_user_checkin_value(user_id)
     when VERIFIED_ROLE_ID
       role_yaml_id = "checkin_verified"
     when INVALID_ROLE_ID
-      role_yaml_id = "checkin_new"    
+      role_yaml_id = "checkin_new"
     end
 
     if role_yaml_id == nil
-      raise RuntimeError, "Unexpected role ID received, there may be a new role that needs to be accounted for by checkin!"
```

Comment:
please revert line 70 to `raise RuntimeError, "Unexpected role ID received, there may be a new role that needs to be accounted for by checkin!"`

Reply:
ok